src/hooks/useMetamask.ts

```diff
@@ -10,20 +10,25 @@ export const injected = new InjectedConnector({
   supportedChainIds: [1, 56]
 });
 
-export function useEagerConnect(isInactive) {
+export function useEagerConnect(isInactive, isInterval) {
   const web3React = useWeb3React();
   const { pathname } = useLocation();
   const [chainInfo] = useGlobalState('chainInfo');
   const [metamaskAddress, setMetamaskAddress] =
     useGlobalState('metamaskAddress');
 
   useEffect(() => {
+    if(isInterval) return;
     eagerConnectBsc();
   }, [pathname]);
 
   const eagerConnectBsc = () => {
```

Comment:
should change to **eagerConnectEvm** then check chainId is default Bsc or ethereum